src/app/containers/ATIAnalytics/params/index.js

```diff
@@ -37,6 +41,8 @@ const pageTypeUrlBuilders = {
   mostWatched: buildMostWatchedATIUrl,
   IDX: buildIndexPageATIUrl,
   FIX: buildIndexPageATIUrl,
+  TOPIC: (data, requestContext, serviceContext) =>
+    buildTopicPageATIUrl(data, requestContext, serviceContext),
```

Comment:
I believe this wrapping code has no effect
```suggestion
  TOPIC: buildTopicPageATIUrl,
```